src/routes/_layout/contact.tsx

```diff
@@ -1,10 +1,181 @@
-import { PageUnderConstruction } from "@/components/page-under-construction";
+import { useState, useEffect, useRef } from "react";
 import { createFileRoute } from "@tanstack/react-router";
+import { FaArrowRightLong } from "react-icons/fa6";
+import emailjs from "@emailjs/browser";
+import "@/styles/contact.css";
 
 export const Route = createFileRoute("/_layout/contact")({
   component: RouteComponent,
 });
 
 function RouteComponent() {
-  return <PageUnderConstruction />;
+  // emailjs variables, fill in when make account on gmc gmail
+  const service_id = "";
+  const template_id = "";
+  const public_key = "";
```

Comment:
What is this, Python?! lol

```suggestion
  const SERVICE_ID = "";
  const TEMPLATE_ID = "";
  const PUBLIC_KEY = "";
```